iceoryx_posh/include/iceoryx_posh/internal/gateway/gateway_generic.inl

```diff
@@ -1,5 +1,6 @@
 // Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
 // Copyright (c) 2020 - 2021 by Apex.AI Inc. All rights reserved.
+// Copyright (c) 2022 by NXP. All rights reserved.
```

Comment:
@JakubSosnovec NXP, like the semiconductor company?

Reply:
That one, indeed :) we also do software to run on those semiconductors.

Reply:
Cool. A few years ago I was also designing PCBs and used quite a lot of NXP products :)

Reply:
Very nice! Hopefully, they served you well :)

To be honest, working for a semiconductor vendor doesnt necessarily mean understanding anything about hardware design, which I am a bright example of 😄 .

Thanks a lot to you guys for keeping this stuff open source!